scripts.js

```diff
@@ -0,0 +1,14 @@
+function add_doggo() {
+    var src = "https://dog.ceo/api/breed/shiba/images";
+    show_image("https://dog.ceo/api/breed/shiba/images", 500,500, "Doggo");
+}
+
+
+function show_image(src, width, height, alt) {
+    var img = document.createElement("img");
+    img.src = src;
```

Comment:
If you look at that url, `https://dog.ceo/api/breed/shiba/images`, it isn't an image, it's a JSON response.  You'll need an extra step that parses the API response and assigns it to the image.